content/applications/productivity/discuss/advanced/email_servers.rst

```diff
@@ -224,6 +224,20 @@ alias in your mail server.
 
 .. _Office 365 documentation:
     https://support.office.com/en-us/article/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-office-365-69f58e99-c550-4274-ad18-c805d654b4c4
+    
+Set up different outgoing servers in a multi-company environment
```

Comment:
Hello @tiku-odoo Thanks for the review, looks fine !

Reply:
Thanks-

Can you explain what happens if no servers are returned after the query? For example, two servers are set with from filter email address, how will the transactional email be sent out? 

Our current documentation states that Odoo will use two system parameters to set the "from" address of this email.

- mail.default.from: local part of default outgoing email address.
- mail.catchall.domain: domain part of default outgoing email address. 
`
With this configuration in place, if Odoo sends an email that doesn’t match any of the from filters, it will alter the email’s “From” before sending it to the MTA.

It will use the default outgoing email address, composed like this: {mail.default.from}@{mail.catchall.domain}.
`
https://www.odoo.com/documentation/saas-15.1/applications/general/email_communication/email_servers.html#use-a-default-from-email-address


Thanks for clarifying. 

Tim

Reply:
Hello @tiku-odoo 

Here is what is done in the code : https://github.com/odoo/odoo/blob/1b9dd118cb0f305ed38b6326fa814336cbf16252/odoo/addons/base/models/ir_mail_server.py#L598

Basically, if there are no email server corresponding to the "From filter", the first email server found is used even if it was configured for another domain.

If there is no email server, the email will never be sent, just like in SaaS when you duplicate a database : the standard email server from Odoo is archived with the aim to avoid sending fake invoices to real customers during the testing process.

Hope this helps.